src/main/resources/assets/iris/lang/lol_us.json

```diff
@@ -1,33 +1,33 @@
 {
-  "iris.shaders.reloaded": "Shadews wewoaded!",
-  "iris.shaders.toggled": "Toggwed shadews to %s!",
-  "iris.keybind.reload": "Wewoad shadews",
-  "iris.keybind.shaderPackSelection": "Shadewpack sewection scween",
-  "iris.keybind.toggleShaders": "Toggwe Shadews",
-  "iris.keybinds": "Iwis",
-  "iris.shaders.reloaded.failure": "Faiwed to wewoad shadews! Reason: %s",
-  "iris.shaders.toggled.failure": "Faiwed to toggwe shadews! Reason: %s",
-  "iris.sodium.failure.title": "Iwis faiwed to woad!",
-  "iris.sodium.failure.reason": "You have Sodium in youw mods fowdew, howevew Iwis incwudes it. Pwease cwose da game, and wemove Sodium from youw mods fowdew.",
+  "iris.shaders.reloaded": "Shaedews wewoded!",
+  "iris.shaders.toggled": "Turnd Blek cat un grownd to %s!",
+  "iris.keybind.reload": "Rewoaded Blek cat un gownd",
+  "iris.keybind.shaderPackSelection": "Blek Ket On Grownd Bramd Selectur scren",
+  "iris.keybind.toggleShaders": "Toggwe Blek Cat Un Ground",
+  "iris.keybinds": "Iris",
+  "iris.shaders.reloaded.failure": "COWLDNT REWOAD BLECK KET ON GROWND! Reson: %s",
+  "iris.shaders.toggled.failure": "COWLDNT FIND BLECK KET TO PUT UN GROWND! Reson: %s",
+  "iris.sodium.failure.title": "Iris cowld nut be able to woad!",
+  "iris.sodium.failure.reason": "You hab Sodium in kiten's mods fowder, however Iris includes it. Pleas cwose da game, and rembve Sodium from kitten's mods fowder.",
 
-  "options.iris.apply": "Appwy",
-  "options.iris.refresh": "Refwesh",
-  "options.iris.openShaderPackFolder": "Open Shadew Pack Fowdew...",
-  "options.iris.refreshShaderPacks": "Refwesh Shadew Packs",
-  "options.iris.shaderPackSelection": "Shadew Packs...",
-  "options.iris.shaderPackSelection.title": "Shadew Packs",
-  "options.iris.shaderPackSelection.addedPack": "Added Shadew Pack \"%s\"",
-  "options.iris.shaderPackSelection.addedPacks": "Added %s Shadew Packs",
-  "options.iris.shaderPackSelection.failedAdd": "Fiwes wewe not vawid Shadew Packs",
-  "options.iris.shaderPackSelection.failedAddSingle": "\"%s\" ish not a vawid Shadew Pack",
+  "options.iris.apply": "DUN YET?!",
+  "options.iris.refresh": "Reflesh",
+  "options.iris.openShaderPackFolder": "Open Blek Ket Veriants Fowder...",
```

Comment:
"see yr shadurz foldur"
"mak yr shadur pakz fresh"
"Shadur Packz..." (consistency with resource pack button)
"Shadur Packz"
"I can has shadur pack \"%s\""
"I can has %s shadur packz"
"Dees rnt reel shadur packz!!"
"\"%s\" iznt a reel shadur pack!!"
"Cant hav dis shadur pack"

Hopefully keep the theme for other names

Reply:
very